src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonObjectCreationHandlingAttribute.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json.Serialization;
+
+/// <summary>
+/// Determines how deserialization will handle object creation for properties.
+/// </summary>
+/// <remarks>
+/// When placed on a field or property, indicates if member will replaced or populated
+/// and when default resolvers are used this will be mapped to <see cref="JsonPropertyInfo.ObjectCreationHandling"/>.
```

Comment:
Seems like this should be a separate sentence. Also, consider clarifying what "default resolvers" mean in this context.
```suggestion
/// When default resolvers are used this will be mapped to <see cref="JsonPropertyInfo.ObjectCreationHandling"/>.
```

Reply:
will be fixed in the next operation